src/api.js

```diff
@@ -0,0 +1,74 @@
+const form = document.getElementById('form');
+const gameId = localStorage.getItem('gameId');
+
+async function submitForm(event) {
```

Comment:
- [ ] As suggested by the previous reviewer, part of the requirement for this milestone is to use ES6 arrow functions, I'll suggest you rewrite this function to use arrow function syntax. :+1: :+1:  IE: `const submitForm = async (event) => { };` :smile: :small_airplane: